logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -183,12 +183,24 @@ public void open() throws IOException {
             for (int pageNum = headCheckpoint.getFirstUnackedPageNum(); pageNum < headCheckpoint.getPageNum(); pageNum++) {
 
                 // all tail checkpoints in the sequence should exist, if not abort mission with a NoSuchFileException
-                Checkpoint cp = this.checkpointIO.read(this.checkpointIO.tailFileName(pageNum));
+                final Checkpoint cp = this.checkpointIO.read(this.checkpointIO.tailFileName(pageNum));
```

Comment:
nit: cp --> checkpoint ? (like other variables in the this class)

Reply:
meh, will pass on this one ;)